# Compute next pose state

# Predict prior belief
sim_odom_topic: /sim/odom
sim_velocity_cmd_topic: /sim/cmd_vel
sim_controller_name: /sim/move_base
sim_odom_frame: sim/odom
sim_robot_base_frame: sim/base_footprint
sim_set_pose_topic: /sim/set_pose
odom_topic: /odometry/filtered

# Send rendezvous pose state
rendezvous_belief_topic: rendezvous_belief

# Move to rendezvous pose state
controller_name: move_base

# Listen for measurement state
measurement_from_viewer_topic: pose_measurements
listen_wait_timeout: 30
listen_max_messages: 5
listen_delay: 5.0

# Update belief state
localizer_measurement_topic: global_pose
